LearningMissionLab/WorkExperience.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LearningMissionLab
+{
+    public class WorkExperience
+    {
+        DateTime _startDate;
+        DateTime _endDate;
+        string _companyName;
+        string _position;
+        string _cityName;
+        List<Project> _projectList;
+
+        public WorkExperience() {}
+        
+        public WorkExperience
+        (
+            DateTime startDate,
+            DateTime endDate,
+            string companyName,
+            string position,
+            string cityName,
```

Comment:
you should use contactInfo instead of cityName.

Reply:
Done.